tests/test_headers.py

```diff
@@ -336,6 +336,16 @@ def test_response_header_helpers_on_get(self):
         self.assertIn(('content-range', 'bytes 0-499/10240'),
                       self.srmock.headers)
 
+        resp.content_range = (0, 499, 10 * 1024, 'bytes')
+        self.assertEqual('bytes 0-499/10240', resp.content_range)
+        self.assertIn(('content-range', 'bytes 0-499/10240'),
+                      self.srmock.headers)
+
+        resp.content_range = (0, 25, 100, 'items')
+        self.assertEqual('items 0-25/100', resp.content_range)
+        self.assertIn(('content-range', 'bytes 0-499/10240'),
```

Comment:
Thank you for your feedback. You are right, the second unit test should be testing for `items 0-25/100`. I have fixed this in unit test and also modified the mock response headers generator to return other range units than `bytes` if requested.